tests/ecmp/test_fgnhg.py

```diff
@@ -173,8 +178,94 @@ def partial_ptf_runner(ptfhost, test_case, dst_ip, exp_flow_count, **kwargs):
             qlen=1000,
             log_file=log_file)
 
+def hash_view_cli_verification(duthost, prefix_list):
+    header = ["FG NHG Prefix", "Next Hop", "Hash buckets"]
+    table = []
+    mode = ""
+    compressed_ipv6_prefixes = [] 
+
+    if ":" in prefix_list[0]:
+        mode = "fgnhg_v6" 
+        for ipv6_prefix in prefix_list:
+            subnet = ipv6_prefix.split("/")[1]
+            ipv6_prefix = ipv6_prefix.split("/")[0]
+            ipv6_prefix = unicode(str(ipaddress.IPv6Address(ipv6_prefix)) + "/" + subnet)
```

Comment:
@kktheballer this looks a bit of overhead...consider using `ipaddress.ip_interface`

Reply:
done

Reply:
Hi Nazarii, this long way of doing it ensures that we get a compressed form of ipv6 address .. because sometimes the extra 0's throws the whole code flow off

Reply:
@kktheballer please provide an example

Reply:
>>> ipaddress.ip_address('2001:db8::00')  --> IPv6Address('2001:db8::') 

(The 0s get truncated here to condense the ip address. I will do for all instances of IPV6 prefix so that we can str-compare them properly)